vendor/k8s.io/kubernetes/cmd/kubelet/app/server_patch.go

```diff
@@ -0,0 +1,91 @@
+package app
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/golang/glog"
+	yaml "gopkg.in/yaml.v2"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/kubernetes/pkg/volume"
+	"k8s.io/kubernetes/pkg/volume/emptydirquota"
+)
+
+// VolumeConfig contains options for configuring volumes on the node.
+type VolumeConfig struct {
```

Comment:
You'll want a typemeta here for versioning this format.

Reply:
typemeta.  what is this?